src/system/ZAuthModule/Resources/views/Authentication/EmailLoginBlock.topnav.html.twig

```diff
@@ -1,5 +1,5 @@
 {% form_theme form with [
-'ZikulaFormExtensionBundle:Form:bootstrap_3_zikula_admin_layout.html.twig',
+'bootstrap_3_layout.html.twig',
```

Comment:
@Guite this has already been merged, but I do not like this solution. because it removes other customizations that are important to the UI. for example, the icons are now missing:

![screen shot 2016-09-10 at 4 08 46 pm](https://cloud.githubusercontent.com/assets/350048/18413270/ea8cca66-7770-11e6-907f-da634a60a42a.png)

has become

![screen shot 2016-09-10 at 4 09 00 pm](https://cloud.githubusercontent.com/assets/350048/18413271/efcbb712-7770-11e6-8693-99ddb4f234ca.png)

of course the checkbox issue is fixed, but IMO, this is the wrong way to fix it.